src/OrbitGl/EventTrack.cpp

```diff
@@ -32,8 +32,10 @@ using orbit_client_protos::CallstackEvent;
 namespace orbit_gl {
 
 EventTrack::EventTrack(OrbitApp* app, TimeGraph* time_graph, TimeGraphLayout* layout,
-                       const CaptureData* capture_data, ThreadID thread_id)
-    : ThreadBar(app, time_graph, layout, capture_data, thread_id), color_{0, 255, 0, 255} {}
+                       const CaptureData* capture_data, ThreadID thread_id,
+                       CaptureViewElement* parent)
+    : ThreadBar(app, time_graph, layout, capture_data, thread_id, parent, "Events"),
```

Comment:
Seeing `"Events"` passed as `name` and used in the E2E tests reminded me that `CallstackTrack` and `"Callstacks"` would be more appropriate. Just throwing it there...

Reply:
I was going to rename those classes to not be called "Track" anymore, I think `CallstackThreadBar` would be more fitting given the base class is called `ThreadBar`. But I wanted to do this in a follow-up PR to make the review easier.